### DISCUSSION TITLE
2023-06-27 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#273] [Meetings]
### Date: 2023-06-27, 3:00 – 4:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
| 3:00 | Welcome | Marjorie Rallins |
| 3:05 - 3:30 | Turbidity | William Hess, William Smith |
| 3:30 - 3:35 | Other proposal items | Eza Hafeza |
| 3:35 - 4:00 | Governance | Marjorie Rallins, Virginia Riehl |
| 4:00 | Adjourn | Marjorie Rallins |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
|[GitHub #180] Proposal k[UA]/l  - advising current use of U in UCUM standard; invite requestor to follow and/or engage in the GitHub issue | 2022-10-03 | 2023-05-27 presentation; awaiting decision | Eza Hafeza |
|[GitHub #182] Chemiluminescence Units – Request more info from the submitter: context, relativeness to base units|2022-10-03| On hold |Eza Hafeza|
|GitHub [multiple unit requests](https://github.com/ucum-org/ucum/issues?q=is%3Aissue+is%3Aopen+turbidity) Turbidity – Request more information from submitter: context, relativeness to base units|2022-10-03| 2023-06-27 presentation, William Smith |Eza Hafeza|

***LINKS TO REFERENCE MATERIALS***
[Cooperation with CIPM on Digital-SI.ProvBy Simon COx](https://drive.google.com/file/d/159QJncutSKxj3FT06uxTzoAcRC5tKvUi/view?usp=share_link)
[All open issues on UCUM GitHub](https://github.com/ucum-org/ucum/issues/200)
[UCUM Committee_Meeting Summary_2023-05-23_Final.docx](https://docs.google.com/document/d/1B13KO6orkCFJC2XYZRS_PEeOE1VTSUbn/edit)
[IgE references](https://drive.google.com/drive/folders/10E4AS8cGLYnh1vkrIhloUQUKjH9KafGW)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by dr-shorthair** [comment]
I will be unable to attend this meeting as it is 5am local time.

**Comment by clemmcdonald** [comment]
Under stood. We should be adjusting to a time that is reasonable for you

From: Simon Cox ***@***.***>
Sent: Tuesday, June 27, 2023 6:40 AM
To: ucum-org/ucum ***@***.***>
Cc: McDonald, Clem (NIH/NLM/LHC) [E] ***@***.***>; Team mention ***@***.***>
Subject: [EXTERNAL] Re: [ucum-org/ucum] 2023-06-27 Meeting Agenda (Discussion #273)


I will be unable to attend this meeting as it is 5am local time.

-
Reply to this email directly, view it on GitHub<https://github.com/orgs/ucum-org/discussions/273#discussioncomment-6292202>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3VBCFNS47BTAYH2LW6VWFDXNK2BFANCNFSM6AAAAAAZUUIT7M>.
You are receiving this because you are on a team that was mentioned.Message ID: ***@***.******@***.***>>
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and are confident the content is safe.
